configurations/performance/cassandra_stress_gradual_load_reduced_steps_number.yaml

```diff
@@ -4,5 +4,5 @@
 ##   - a single-element list or integer (applied to all throttle steps)
 ##   - a list with the same length as perf_gradual_throttle_steps[load] (one thread count per step).
 perf_gradual_threads: {"read": 620, "write": 400, "mixed": 1900}
-perf_gradual_throttle_steps: {"read": ['450000', '700000', 'unthrottled'], "mixed": ['300000', '450000', 'unthrottled'], "write": ['200000', 'unthrottled']}  # where every value is in ops
+perf_gradual_throttle_steps: {"read": ['450000', '700000', 'unthrottled'], "mixed": ['300000', '450000', 'unthrottled'], "write": ['200000', 'unthrottled'], "read_disk_only": ['250000', '300000', 'unthrottled']}  # where every value is in ops
 perf_gradual_step_duration: {"read": '20m', "write": None, "mixed": '20m'}
```

Comment:
don't you need here also read_disk_only ?

Reply:
fixed